packages/components/app/styles/components/pagination.scss

```diff
@@ -166,3 +166,25 @@ $pagination-height: calc(36px + 6px);
     background-position: center right 5px;
   }
 }
+
+// Responsive layout
+// max-width is based on where component layout breaks down currently, not finalized
```

Comment:
Note: If we want to keep these responsive layout styles then probably I would update the break point to use the “official” small screen size break point. The current one is just a loose experimental value. 

(I used a regular media query vs. a container query as the container query would require adding an extra wrapper div to apply styling to the container plus browser support is still pretty recent.)

Reply:
There are no "official" breaking points, unfortunately.

Reply:
I understand, that's why I put quotes :) 

If there is something we are using in the website or something loose that's been discussed though I thought I could try to follow that if it made sense.